README.md

```diff
@@ -14,35 +14,20 @@ We provide a GitHub Action.
 ```
 
 This extracts a pre-built MLIR installation, adds the binaries to `$PATH`, and defines `$LLVM_DIR` and `$MLIR_DIR`.
-The pre-built MLIR installation can be found in the release assets on GitHub.
-We are using calendar versioning for our releases.
-Refer to the release notes of the respective release for information on the LLVM version and the build settings.
 
 ## Installation Scripts
 
 If you want to use the pre-built MLIR installations locally, we also provide installation scripts.
 The scripts require the LLVM version (e.g., `21.1.7`) and the desired installation directory to be passed.
 
-On Linux and macOS, use the following to run the installation script:
+On Linux and macOS, use the following Bash command:
 
 ```bash
 curl -LsSf https://github.com/munich-quantum-software/setup-mlir/releases/latest/download/setup-mlir.sh | bash -s -- -v 21.1.7 -p /path/to/installation
 ```
 
-On Windows, use this PowerShell command:
+On Windows, use the following PowerShell command:
 
 ```powershell
 powershell -ExecutionPolicy ByPass -c "irm https://github.com/munich-quantum-software/setup-mlir/releases/latest/download/setup-mlir.ps1 | iex; setup-mlir.ps1 -llvm_version 21.1.7 -install_prefix /path/to/installation"
```

Comment:
As I am currently debugging on Windows, I noticed that this command would not run.
The thing I got to work was

```suggestion
powershell -ExecutionPolicy ByPass -c "& ([scriptblock]::Create((irm https://github.com/munich-quantum-software/setup-mlir/releases/latest/download/setup-mlir.ps1))) -llvm_version 21.1.8 -install_prefix /path/to/installation"
```
(initially the above also failed; but that was due to me not having zstd installed)